server/user.go

```diff
@@ -250,6 +250,8 @@ func (p *Plugin) connectUser(instance Instance, mattermostUserID types.ID, conne
 		return err
 	}
 
+	_ = p.setupFlow.ForUser(string(mattermostUserID)).Go(stepConnected)
```

Comment:
Does this error if there is no flow running for the user?

Reply:
Good catch! fixing now.